Source/Scene/GroundPolylineBatch.js

```diff
@@ -0,0 +1,498 @@
+        if (defined(primitive._rs)) {
+            return;
+        }
+
+        var polygonOffset = {
+            enabled : true,
+            factor : -5.0,
+            units : -5.0
+        };
+
+        primitive._rs = RenderState.fromCache({
+            blending : BlendingState.ALPHA_BLEND,
+            depthMask : false,
+            depthTest : {
+                enabled : true
+            },
+            polygonOffset : polygonOffset
+        });
+
+        primitive._rsPick = RenderState.fromCache({
+            depthMask : false,
+            depthTest : {
+                enabled : true
+            },
+            polygonOffset : polygonOffset
+        });
+    }
+
+    var PolylineFS =
+        'uniform vec4 u_highlightColor; \n' +
+        'void main()\n' +
+        '{\n' +
+        '    gl_FragColor = u_highlightColor;\n' +
+        '}\n';
+
+    function createShaders(primitive, context) {
+        if (defined(primitive._sp)) {
+            return;
+        }
+
+        var batchTable = primitive._batchTable;
+
+        var vsSource = batchTable.getVertexShaderCallback(false, 'a_batchId')(GroundPolylineBatchVS);
+        var fsSource = batchTable.getFragmentShaderCallback()(PolylineFS);
+
+        var vs = new ShaderSource({
+            defines : ['VECTOR_TILE'],
+            sources : [PolylineCommon, vsSource]
+        });
+        var fs = new ShaderSource({
+            defines : ['VECTOR_TILE'],
+            sources : [fsSource]
+        });
+
+        primitive._sp = ShaderProgram.fromCache({
+            context : context,
+            vertexShaderSource : vs,
+            fragmentShaderSource : fs,
+            attributeLocations : attributeLocations
+        });
+
+        vsSource = batchTable.getPickVertexShaderCallback('a_batchId')(GroundPolylineBatchVS);
+        fsSource = batchTable.getPickFragmentShaderCallback()(PolylineFS);
+
+        var pickVS = new ShaderSource({
+            defines : ['VECTOR_TILE'],
+            sources : [PolylineCommon, vsSource]
+        });
+        var pickFS = new ShaderSource({
+            defines : ['VECTOR_TILE'],
+            sources : [fsSource]
+        });
+        primitive._spPick = ShaderProgram.fromCache({
+            context : context,
+            vertexShaderSource : pickVS,
+            fragmentShaderSource : pickFS,
+            attributeLocations : attributeLocations
+        });
+    }
+
+    function queueCommands(primitive, frameState) {
+        if (!defined(primitive._command)) {
+            var uniformMap = primitive._batchTable.getUniformMapCallback()(primitive._uniformMap);
+            primitive._command = new DrawCommand({
+                owner : primitive,
+                vertexArray : primitive._va,
+                renderState : primitive._rs,
+                shaderProgram : primitive._sp,
+                uniformMap : uniformMap,
+                boundingVolume : primitive._boundingVolume,
+                //pass : Pass.GROUND
```

Comment:
Does this need a `TODO` so we add the `GROUND_TRANSLUCENT` pass before merging `3d-tiles` into master?